### PR TITLE
Upgrade jackson library from 1.8 to 2.10

### PR DESCRIPTION
**Pull request by justinlin-linkedin**
Jackson 1.8 is pretty old

### CODE REVIEW

File: ambry-quota/src/integration-test/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresherTest.java [hunk 1]
```diff
@@ -43,8 +44,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Function;
-import org.codehaus.jackson.map.ObjectMapper;
-import org.codehaus.jackson.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
```
Comment (jsjtzyy) [inline ambry-quota/src/integration-test/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresherTest.java:47]: nit: rearrange the imports

File: build.gradle [hunk 2]
```diff
@@ -195,6 +195,7 @@ project(':ambry-account') {
                 project(':ambry-mysql')
         compile ("org.apache.helix:helix-core:$helixVersion") {
             exclude group: "log4j", module: "log4j"
+            exclude group: "org.codehaus.jackson"
```
Comment (jsjtzyy) [inline build.gradle:198]: just to confirm, did you double check the version of jackson that is adopted by Helix lib?

**Review by jsjtzyy — APPROVED** [review]
LGTM

File: build.gradle (hunk 2 above)
Comment (justinlin-linkedin) [reply build.gradle]: i remove this exclude statements. It causes some test failures.

**Pull request merged**